packages/orbit-components/src/InputField/types.d.ts

```diff
@@ -41,14 +41,14 @@ export interface Props extends Common.Globals, Common.SpaceAfter, Common.DataAtt
   readonly id?: string;
   readonly insideInputGroup?: boolean;
   readonly helpClosable?: boolean;
-  readonly onChange?: InputEvent;
-  readonly onFocus?: InputEvent;
-  readonly onBlur?: InputEvent;
+  readonly onChange?: React.ChangeEventHandler<HTMLInputElement>;
+  readonly onFocus?: React.FocusEventHandler<HTMLInputElement>;
+  readonly onBlur?: React.FocusEventHandler<HTMLInputElement>;
   readonly onSelect?: InputEvent;
```

Comment:
Is this expected to be kept with this type?

Reply:
what event do u suggest to use instead?) for onSelect only SyntheticEvent

Reply:
I didn't check if there was a SelectEventHandler, just wanted to make sure this was intentional 👍